ui/src/shared/utils/debouncer.ts

```diff
@@ -0,0 +1,43 @@
+type F = (...args: any[]) => any
```

Comment:
could you name this something that is clearer? even `Func` if that's what it means?